modin/pandas/base.py

```diff
@@ -2863,6 +2863,32 @@ def tz_localize(
         )
         return self.set_axis(labels=new_labels, axis=axis, inplace=not copy)
 
+    # TODO: uncomment the following lines when #3331 issue will be closed
+    # @prepend_to_notes(
+    #     """
+    #     In comparison with pandas, Modin's ``value_counts`` returns Series with ``MultiIndex``
+    #     only if multiple columns were passed via the `subset` parameter, otherwise, the resulted
+    #     Series's index will be a regular single dimensional ``Index``.
+    #     """
+    # )
+    # @_inherit_docstrings(pandas.DataFrame.value_counts, apilink="pandas.DataFrame.value_counts")
+    def value_counts(
+        self,
+        subset: Sequence[Hashable] = None,
+        normalize: bool = False,
+        sort: bool = True,
+        ascending: bool = False,
+        dropna: bool = True,
+    ):
+        if subset is None:
+            subset = self._query_compiler.columns
+        counted_values = self.groupby(by=subset, sort=False, dropna=dropna).size()
+        if sort:
+            counted_values.sort_values(ascending=ascending, inplace=True)
+        if normalize:
+            counted_values = counted_values / counted_values.sum()
```

Comment:
Slight style preference here:
```suggestion
            return counted_values / counted_values.sum()
```

Reply:
can't agree with this because there potentially could be some [extra logic of postprocessing the final result](https://github.com/dchigarev/modin/blob/e1d93eea74cac0f959bc920270b6327cd3b8c194/modin/pandas/base.py#L2890-L2895)